packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -516,6 +516,20 @@
                 }
             ]
         },
+        {
+            "name": "stepfunctions_saveFileFromWorkflowStudio",
+            "description": "Called after the user saves or updates the file in Step Functions Workflow Studio custom editor",
```

Comment:
Can you create an action item to upstream this to aws-toolkit-common after release?

Reply:
I have already created a task in the project to move the new metrics (since this question was raised in the previous pull request)